<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="1" hostname="sampritis-MacBook-Pro.local" failures="1" tests="3" name="Personal Loan" time="0.25" errors="0" timestamp="2024-04-24T22:08:32 IST">
  <testcase classname="test.day1" name="Demo" time="0.04">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at test.day1.Demo(day1.java:24)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 21 stack frames]]>
    </failure>
  </testcase> <!-- Demo -->
  <testcase classname="test.day2" name="ploan" time="0.006"/>
  <testcase classname="test.day4" name="NewtravelLoan" time="0.005"/>
  <testcase classname="test.day1" name="SecondTest" time="0.0">
    <ignored/>
  </testcase> <!-- SecondTest -->
</testsuite> <!-- Personal Loan -->
